permission_handler/example/pubspec.yaml

```
@@ -15,7 +15,7 @@ dev_dependencies:
   permission_handler:
     path: ../
 
-  url_launcher: ^5.4.11
+  url_launcher: ^6.0.0-nullsafety.6
```

Can be updated to the stable version:
```suggestion
  url_launcher: ^6.0.0
```